### PR TITLE
Provide diagram of k8gb internal components

### LINKED ISSUE #89: Document internal components of k8gb
We already have great design architecture docs in place, now we need to amend them with specific implementation details depicting open source components we are using to solve specific tasks, e.g.

* CoreDns to respond to DNS queries
* Dedicated ETCD to act as CoreDNS backend
* external DNS as a glue between DNSEndpoint CRDs and CoreDNS configuration
* Set of CRDs and k8s primitives that involved in k8gb operations 

Fancy diagrams are welcomed.

### PR DESCRIPTION
**Pull request by ytsarev**
* Resolves #89
* Inline production readiness table update

### CODE REVIEW

**Review by donovanmuller — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -75,12 +77,12 @@ Please see the extended architecture documentation [here](/docs/index.md)
...
 
-| Type                            | Implementation                                     |
-|---------------------------------|----------------------------------------------------|
-| Kubernetes Version              | >= 1.14 (with install workaround) >= 1.15 (Stable) |
-| Ingress Controller              | Nginx                                              |
-| EdgeDNS                         | Infoblox                                           |
-| Number of k8gb enabled clusters | 2                                                  |
+| Type                             | Implementation                                     |
+|----------------------------------|----------------------------------------------------|
+| Kubernetes Version               | >= 1.14 (with install workaround) >= 1.15 (Stable) |
```
Comment (somaritane) [inline README.md:82]: Not in the scope, just as you change it, makes sense to provide the reference to install workaround below the table.

**Review by somaritane — DISMISSED** [review]
lgtm!

File: README.md (hunk 1 above)
Comment (ytsarev) [reply README.md]: @somaritane good point, done, please check https://github.com/AbsaOSS/k8gb/pull/186/commits/fa07ac0b86d03c1932f229cc124d11f0c09259f5

Comment (somaritane) [reply README.md]: Looks good!

**Review by somaritane — APPROVED** [review]

**Review by donovanmuller — APPROVED** [review]

**Pull request merged**